c++/src/capnp/rpc.c++

```diff
@@ -210,24 +229,48 @@ public:
     }
   }
 
+  T& nextHigh(Id& id) {
+    // Choose an ID with the top bit set in round-robin fashion, but don't choose an ID that
+    // is still in use.
+
+    bool created = false;
+    T* slot;
+    while (!created) {
```

Comment:
<img src="https://i.kym-cdn.com/photos/images/original/001/365/753/94c.jpg" width="300" alt="I guess">

This would probably require more RAM than anyone has but I suppose I could check for it...